src/ui.jl

```diff
@@ -42,7 +42,7 @@ function TerminalMenus.header(m::CthulhuMenu)
     m.sub_menu && return ""
     """
     Select a call to descend into or ↩ to ascend. [q]uit.
-    Toggles: [o]ptimize, [w]arn, [d]ebuginfo.
+    Toggles: [o]ptimize, [w]arn, [d]ebuginfo, [s]yntax highlight for LLVM/Native.
     Show: [L]LVM IR, [N]ative code
```

Comment:
I didn't use `[h]ighlight` as it may make sense to use it for `[h]elp` at some point. I guess another option would be `Show: [L]LVM IR, [N]ative code (toggle: [H]ighlight)`.